Tests/SentryProfilerTests/SentryProfilerSwiftTests.swift

```diff
@@ -11,7 +12,7 @@ class SentryProfilerSwiftTests: XCTestCase {
             options.dsn = SentryProfilerSwiftTests.dsnAsString
             return options
         }()
-        lazy var client: TestClient = TestClient(options: options)!
+        lazy var client: TestClient? = TestClient(options: options)
```

Comment:
No test logic changes here; I converted everything to avoid force-unwrapping to avoid crashing the test process.